Add Ubuntu-themed header

s-makin opened the pull request:
```
This change will further align us with Ubuntu branding.

I haven't changed much past the default in the starter pack
so I can tailor it to our needs. Let me know what other links
we should add to the "more resources" menu, or any other changes
you'd like me to make. When I uprate the devdocs, they will be
added to this header (so they align with ubuntu.com docs)
```

## Does this PR require extra reviews?
<!-- Should people outside of the team see and approve these changes before the
PR gets merged? If yes, make sure to tag them as reviewers. -->
 - [ ] Yes
 - [X] No

s-makin commented:
> @s-makin just a question, do we need to keep the link around in the header ? Because if we click on the logo it already redirects you to the right place right ?
> 
> My position would be to remove it or make sure that it is highlighted somewhere else in the landing page

No, I don't think we need to - actually my intention would be to replace it with something more useful to avoid the repetition. I'm just not sure what to replace it with!

lucasmoura commented:
I think having nothing there would be okay right now

orndorffgrant commented:
I think we could put a link to the discourse somewhere in the header too.
Another idea is to have a "Services" dropdown with links to ubuntu.com pages for esm, livepatch, fips etc.

Also there is an extra list item in the "More resources" dropdown above "Github" which is empty

s-makin commented:
> I think we could put a link to the discourse somewhere in the header too. Another idea is to have a "Services" dropdown with links to ubuntu.com pages for esm, livepatch, fips etc.
> 
> Also there is an extra list item in the "More resources" dropdown above "Github" which is empty

Great suggestion - thanks Grant! I've added the list of services as suggested. I wasn't sure whether the GitHub link really belonged there after that so I've removed it.
